tinyagentos/routes/coding_sessions.py

```
@@ -0,0 +1,280 @@
+        _launcher(request).stop(session_id)
+    except Exception:  # noqa: BLE001
+        logger.warning("tmux stop failed for %s", session_id, exc_info=True)
+    updated = await store.set_status(session_id, "stopped")
+    return updated
+
+
+@router.post("/api/coding-sessions/{session_id}/archive")
+async def archive_coding_session(
+    session_id: str,
+    request: Request,
+    user: CurrentUser = Depends(current_user),
```

Ruff (0.15.18)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)